addon/templates/components/polaris-card.hbs

```diff
@@ -1,4 +1,4 @@
-{{#if title}}
+{{#if (or title headerActions)}}
   {{#if headerActions}}
```

Comment:
nit: hmm, I guess we're not matching Polaris exactly here, which has this in a [`Header`](https://github.com/Shopify/polaris-react/blob/master/src/components/Card/components/Header/Header.tsx#L29) subcomponent and also checks if it has children

no need to fix this now....can we add a TODO here for that? I guess this entire component is a good candidate for refactoring soon too

Reply:
Yeah exactly, would def want to rebuild this component as and when time allows, it's one from the early days so would be great to bring it in line with the React implementation 🙏